src/components/apps/savedLaunch/SavedLaunchListing.js

```diff
@@ -198,7 +215,24 @@ function ListSavedLaunches(props) {
         queryFn: () => listSavedLaunches({ appId }),
     });
 
+    // It would be better to have a query that is limited to the selected app only
+    const {
+        data: allILs,
+        error: errorILs,
+        isFetching: fetchingILs,
+    } = useQuery(ALL_INSTANT_LAUNCHES_KEY, listFullInstantLaunches);
+
     const savedLaunchClickHandler = (event, savedLaunch) => {
+        var instantLaunch = allILs.instant_launches.find(
```

Comment:
Yeah, may as well be safe with the `?.` -- the saved launches dialog won't render this button until the `allILs` query has returned (since it's using `isQueryLoading([isFetching, fetchingILs])`) but still better than weird errors with undefined values.